spec/javascripts/Navigation/ContextSelector.spec.jsx

```diff
@@ -150,6 +150,11 @@ describe('When there are many services', () => {
 
   it('should mark apis as unauthorized when link is undefined', () => {
     const apiList = contextSelector.find('.PopNavigation-results').children()
-    expect(apiList.find('.unauthorized')).toHaveLength(2)
+    expect(apiList.find('.unauthorized')).toHaveLength(1)
+  })
+
+  it('should render the correct icons for backend and product APIs', () => {
+    const apiIconsClassNames = contextSelector.find('.PopNavigation-results .PopNavigation-link i').map(i => i.prop('className'))
+    expect(apiIconsClassNames).toEqual([ 'fa fa-gift', 'fa fa-gift', 'fa fa-puzzle-piece' ])
```

Comment:
More precisely it should assert that products have the gift, backends the piece. Let me create a small PR if that's ok, since it's not really a big deal.